### PR TITLE
Updated poi config to support eslintignore file

### PR DESCRIPTION
**Pull request by abasic**
- removed old `poi` `eslint` package
 - `ignore` flag is set to true in `poi` config for `eslint-loader`

### CODE REVIEW

File: poi.config.js [hunk 1]
```diff
@@ -101,6 +101,13 @@ module.exports = {
       .plugin('dotenv')
       .use(require.resolve('dotenv-webpack'));
 
+    config
+      .module
+      .rule('eslint')
+      .use('eslint-loader')
+        .loader('eslint-loader')
+        .tap(options => Object.assign(options, { ignore: true }));
```
Comment (vladimyr) [inline poi.config.js]: If I understand code of `eslint-loader` correctly this gets passed down to `eslint`'s `CLIEngine`: https://github.com/webpack-contrib/eslint-loader/blob/2a72daa5b78e3ab0a1d9861210f21e7bb4440948/index.js#L22

That being said, docs on following page: https://eslint.org/docs/developer-guide/nodejs-api#cliengine clearly say that:
>ignore - False disables use of .eslintignore, ignorePath and ignorePattern (default: true). 

So what does this do really? 🤔

Comment (kronicker) [reply poi.config.js]: In build process for production, for some reason, `ignore` would be resolved to `false`, thus _ignoring_ `ignorePath` and/or `ignorePattern`.

Comment (vladimyr) [reply poi.config.js]: Hmm, this thing http://npmfs.com/package/@poi/plugin-eslint/12.0.0/index.js does not toggle it so what does?

Comment (kronicker) [reply poi.config.js]: I know, that's what was baffling me as well.
After more than an hour of debugging, we couldn't figure out what was happening, we only came to the conclusion that setting `ignore: true` explicitly resolves the issue...

Comment (vladimyr) [reply poi.config.js]: I assume you did run `webpack:inspect-config` in prod mode? No luck there?

Comment (kronicker) [reply poi.config.js]: Yes, but the config seemed fine.

**Comment by abasic** [comment]
@vladimyr @kronicker What should we do with this PR?

**Comment by vladimyr** [comment]
> @vladimyr @kronicker What should we do with this PR?

My vote goes for closing it because:
a) we had only one complaint we never fully investigated
b) hopefully #350 will eliminate it :crossed_fingers:

**Pull request closed without merging**